scripts/tests/chiptest/test_definition.py

```diff
@@ -64,17 +64,11 @@ def stop(self):
             return True
         return False
 
-    def reboot(self, discriminator):
-        if self.process:
-            self.stop()
-            self.start(discriminator)
-            return True
-        return False
-
     def factoryReset(self):
-        storage = '/tmp/chip_kvs'
-        if os.path.exists(storage):
-            os.unlink(storage)
+        tempFiles = os.listdir("/tmp")
+        for file in tempFiles:
+            if not os.path.isdir(file) and "chip_kvs" in file:
+                os.unlink("/tmp/" + file)
```

Comment:
This is going to remove unrelated storages, right?  We should be removing the default name unless a different name was passed in and then only removing that passed-in name....

Reply:
So the issue is that we don't know the exact name of the file to remove (this can be specified by the yaml test files to be whatever value with my change). Is it a bad thing to remove all those files that get created on boot up anyway if they're not there. A fresh start on everything?

Reply:
If this is going to be passed in as a parameter we should probably parse the parameter in `__createCommandLineOptions` and save the value somewhere into the app instance so they can be re-used later if needed.

This way you can clear up the "right" thing instead of removing everything which may have side effects.

Reply:
I've stored it in self.kvs of the app instance when calling `start` and deleting only that file during `factoryReset`